scania-digital-design-system/sdds
# feat(side-menu) Migrating side menu

**Describe pull-request**  
Migrating side menu

**Solving issue**  
_In case of GitHub issue add # plus the number of the issue (for example #123) OR if it is Azure then AB# and number of ticket_
Fixes: [AB#2415](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2415)

**How to test**  
1. check out the storybook and see if the side-menu is migrated in accordance with the migration guidelines.
